src/hooks/useCds/index.js

```diff
@@ -5,6 +5,9 @@ import { cdsResources } from 'services/fhir';
 import { valueSetJson } from 'services/valuesets';
 import { translateResponse, translateToggleChange } from './translate';
 import { stridesData } from './strides';
+import { logMsg } from 'util/logger';
+
+const LOGGER_ENABLED = process.env?.REACT_APP_LOGGER_ENABLED || true;
```

Comment:
Should logger default to false, so that using the dashboard right out of the box doesn't necessarily rely on you setting up the logger app? This is minor, and I could go either way on it.

Reply:
Sounds reasonable to me. Changed to false as default